test/CMakeLists.txt

```diff
@@ -46,6 +47,16 @@ foreach (algo_index RANGE 29)# 43
     if (algo_index STREQUAL 10)
       set (enable_ FALSE)
     endif ()
+    # Check if LUKSAN targets are available
+    get_cmake_property(compile_defs_string COMPILE_DEFINITIONS)
+    string(FIND "${compile_defs_string}" "WITH_LUKSAN" has_luksan)
+    if(NOT ${has_luksan} EQUAL TRUE) 
+      if(${algo_index} IN_LIST list_of_algorithms_requiring_luksan)
+      string(CONCAT luksan_missing_msg "Skipping test for algo_index " ${algo_index} " and obj_index " ${obj_index} " because NLOPT is built without option -DNLOPT_WITH_LUKSAN" )
+      message(WARNING ${luksan_missing_msg})
+        set(enable_ FALSE)
+      endif()
+    endif()
```

Comment:
better mark it as skipped at runtime than issuing warnings at configure time using:
`set_tests_properties (xxx PROPERTIES DISABLED TRUE)`

Reply:
Great feedback, thanks! Will push update in a minute.